doc/conf.py

```diff
@@ -67,99 +71,8 @@
 except ImportError:
     print("no netCDF4")
 
-# If extensions (or modules to document with autodoc) are in another directory,
-# add these directories to sys.path here. If the directory is relative to the
-# documentation root, use os.path.abspath to make it absolute, like shown here.
-#sys.path.insert(0, os.path.abspath('.'))
-
-# Monkey patch inspect.findsource to work around a Python bug that manifests on
-# RTD. Copied from IPython.core.ultratb.
-# Reference: https://github.com/ipython/ipython/issues/1456
```

Comment:
see this page for the reference -- it's quite likely that this is no longer an issue now that we've updated ipython for our RTD builds.